src/lapis/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/rbreslow/lua:5.1-alpine
```

Comment:
This container image is derived from the [Lapis docs](https://leafo.net/lapis/reference/getting_started.html#basic-setup) and [`openresty/openresty:alpine-apk`](https://github.com/openresty/docker-openresty/blob/master/alpine-apk/Dockerfile).